openshift/templates/jenkins-ephemeral-monitored.json

```diff
@@ -150,6 +150,10 @@
                   {
                     "name": "JENKINS_UC_INSECURE",
                     "value": "${JENKINS_UC_INSECURE}"
+                  },
```

Comment:
why exposing it?

Reply:
because the casc plugin needs that env var to load the config